src/control/ros/Conversions.cpp

```diff
@@ -281,11 +314,11 @@ std::unique_ptr<SplineTrajectory> toSplineJointTrajectory(
 
   // Map joint indices between jointTrajectory and space subspaces.
   std::map<size_t, size_t> rosJointToMetaSkeletonJoint;
```

Comment:
Unrelated nitpick: I think we could use `std::vector<std::pair<size_t, size_t>>` here instead of `std::map<...>` because we actually don't take the advantage of `std::map` as `std::map::find(...)` is not used. On the contrary, inserting items to `std::map` is more expensive than using `std::vector`.